packages/db-migrations/databases/fxa/patches/patch-136-137.sql

```diff
@@ -0,0 +1,22 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('136');
+
+INSERT INTO securityEventNames(name) VALUES ('account.login.failure');
+INSERT INTO securityEventNames(name) VALUES ('account.two_factor_added');
```

Comment:
We seem to `DELETE FROM` the `account.two_factor_requested` event name below, but not `INSERT INTO` here:
```sql
-- DELETE FROM securityEventNames WHERE name='account.two_factor_requested';
```

Reply:
FWIW, I did Ctrl+F the rest of the event names (above and below) and each seemed to have 4 matches (`INSERT`, `DELETE`, code, and test). 🎉

Reply:
Note, the sql changes are part of the commit this commit is based on.